command/command.go

```diff
@@ -28,16 +29,51 @@ import (
 
 var verbose = false
 
+type LogLevel string
+
+const (
+	INFO LogLevel = "INFO"
+	WARN LogLevel = "WARN"
```

Comment:
If "WARN" is not using now, we can remove the line

Reply:
Removed WARN